cmd/brew-command-not-found-init.rb

```diff
@@ -11,24 +11,29 @@ def help
       # To enable homebrew-command-not-found
       # Add the following lines to ~/.bashrc
 
-      if brew command command-not-found-init > /dev/null; then
-        eval "$(brew command-not-found-init)";
+      HB_CNF_HANDER="$(brew --prefix)/Homebrew/Library/Taps/homebrew/homebrew-command-not-found/handler.sh"
+      if [ -f "$HB_CNF_HANDER" ]; then
+        source "$HB_CNF_HANDER";
       fi
     EOS
   when :fish
     puts <<~EOS
       # To enable homebrew-command-not-found
       # Add the following line to ~/.config/fish/config.fish
 
-      brew command command-not-found-init > /dev/null; and . (brew command-not-found-init)
+      HB_CNF_HANDER="$(brew --prefix)/Homebrew/Library/Taps/homebrew/homebrew-command-not-found/handler.fish"
+      if [ -f "$HB_CNF_HANDER" ]; then
+        source "$HB_CNF_HANDER";
+      fi
```

Comment:
Fish has a different syntax. This should work instead:

```fish
set HB_CNF_HANDER (brew --prefix)"/Homebrew/Library/Taps/homebrew/homebrew-command-not-found/handler.fish"

if test -f $HB_CNF_HANDER
  . $HB_CNF_HANDER
end
```

Reply:
My fault! 😬 I have fixed it and also updated count of formulae & commands. Current count is 17721 commands and 5015 formulae (728 formulae without commands).

Reply:
> P.S. From those 728 formulas, I found some that install cli app or .app bundle -- swift, tarsnap-gui, z, to name a few. Is there a reason why they aren't in the list?

We only track binaries installed under `bin` or `sbin`. `z` installs itself under `etc/profile.d` because it’s a script that, when loaded, creates a command named `z`. This is not an executable file. Swift does have an executable but it’s not installed in a standard location, so even if you `brew install swift` you still need some setup to make it work. We could manually add it to the list, though.